docs/currency.md

```diff
@@ -0,0 +1,159 @@
+# The Currency models
+
+Ledger Live deals with different kind of currencies and assets: fiat currencies, crypto-currencies and tokens.
+They all share similar concept but have specifics.
+
+## The types
+
+The types are defined in [types/currencies](../src/types/currencies.js).
+
+live-common exposes 3 different types: `CryptoCurrency`, `TokenCurrency` and `FiatCurrency`.
+It also exposes a union type `Currency` that is either of these 3 types. _Make sure you use the correct type, sometimes you want the union type Currency, sometimes you are really in context of a CryptoCurrency_.
+
+To ease the differentiation of types, live-common data also introduces a _discriminator field_, `.type`, which have the same name of the type.
+
+### Common fields
+
+Essentially all currencies share this common ground:
+
+- `name: string`: the display name of a currency. For instance `"Bitcoin"` or `"Euro"`.
+- `ticker: string`: the ticker name in exchanges and used for rates (for instance BTC or EUR).
+- `units: Unit[]`: all available `Unit` of the currency (see below what is a Unit). by convention, [0] is the default and have "highest" magnitude.
+- `disableCountervalue?: boolean`: if available, this field express that we should assume there is not valable countervalue for this coin. (because either we can't trade it or it's not available enough to assume enabling its value)
```

Comment:
```suggestion
- `disableCountervalue?: boolean`: if available, this field expresses that we should assume there is not valid counter value for this coin. (because either we can't trade it or it's not available enough to assume enabling its value)
```

I think 🤷‍♂